src/VirtualMonoRepo/source-mappings-ub.json

```diff
@@ -0,0 +1,196 @@
+        },
+        {
+            "name": "cecil",
+            "defaultRemote": "https://github.com/dotnet/cecil"
+        },
+        {
+            "name": "command-line-api",
+            "defaultRemote": "https://github.com/dotnet/command-line-api"
+        },
+        {
+            "name": "deployment-tools",
+            "defaultRemote": "https://github.com/dotnet/deployment-tools"
+        },
+        {
+            "name": "diagnostics",
+            "defaultRemote": "https://github.com/dotnet/diagnostics"
+        },
+        {
+            "name": "emsdk",
+            "defaultRemote": "https://github.com/dotnet/emsdk"
+        },
+        {
+            "name": "format",
+            "defaultRemote": "https://github.com/dotnet/format"
+        },
+        {
+            "name": "fsharp",
+            "defaultRemote": "https://github.com/dotnet/fsharp"
+        },
+        {
+            "name": "installer",
+            "defaultRemote": "https://github.com/dotnet/installer",
+            "exclude": [
+                // We don't need the tarball content in the VMR again, it's already copied in the root
+                "src/SourceBuild/content/**/*"
+            ]
+        },
+        {
+            "name": "msbuild",
+            "defaultRemote": "https://github.com/dotnet/msbuild"
+        },
+        {
+            "name": "nuget-client",
+            "defaultRemote": "https://github.com/NuGet/NuGet.Client",
+            "defaultRef": "dev"
+        },
+        {
+            "name": "razor",
+            "defaultRemote": "https://github.com/dotnet/razor"
+        },
+        {
+            "name": "roslyn",
+            "defaultRemote": "https://github.com/dotnet/roslyn"
+        },
+        {
+            "name": "roslyn-analyzers",
+            "defaultRemote": "https://github.com/dotnet/roslyn-analyzers"
+        },
+        {
+            "name": "runtime",
+            "defaultRemote": "https://github.com/dotnet/runtime"
+        },
+        {
+            "name": "scenario-tests",
+            "defaultRemote": "https://github.com/dotnet/scenario-tests"
+        },
+        {
+            "name": "sdk",
+            "defaultRemote": "https://github.com/dotnet/sdk"
+        },
+        {
+            "name": "source-build-externals",
+            "defaultRemote": "https://github.com/dotnet/source-build-externals",
+            "exclude": [
+                "src/application-insights/**/*.exe",
+                "src/application-insights/**/*.dll",
+                "src/application-insights/**/*.zip",
+                "src/application-insights/**/NuGet.config",
+                "src/humanizer/samples/**/*.js",
+                "src/newtonsoft-json/**/NuGet.Config",
+                "src/spectre-console/docs/**",
+                "src/spectre-console/NuGet.Config",
+                "src/spectre-console/examples/Console/Canvas/Mandelbrot.cs",
+                "src/xunit/NuGet.Config"
+            ]
```

Comment:
Are we still excluding these files intentionally?

Reply:
Yeah, we spoke about this briefly and it is intentional